internal/storage/image.go

```diff
@@ -298,8 +316,25 @@ func (svc *imageService) ImageStatus(systemContext *types.SystemContext, nameOrI
 		return nil, err
 	}
 
-	result := svc.buildImageResult(image, cacheItem)
-	return &result, nil
+	return svc.buildImageResult(systemContext, img, cacheItem, ref)
+}
+
+func inspectInfo(sc *types.SystemContext, ref types.ImageReference) (map[string]string, error) {
+	ctx := context.Background()
+	imgSrcRef, err := ref.NewImageSource(ctx, sc)
```

Comment:
`defer imgSrcRef.Close()` (or perhaps with error checking is missing here.

Reply:
Fixed via https://github.com/cri-o/cri-o/pull/3510#discussion_r402861078